test/lock.test.ts

```diff
@@ -89,4 +89,87 @@ describe("Lock Store Tests", () => {
 
     await expect(store.locks.release(nonAcquiredResourceId, nonAcquiredExecutionId)).rejects.toThrow();
   });
+
```

Comment:
Let's get rid of the tests above, I always felt my implementation here was pretty hacky :D

Reply:
removed.